src/app/share-preview/components/share-preview/share-preview.component.html

```diff
@@ -71,15 +71,18 @@
 <div class="share-preview-content-breadcrumbs">
 	<pr-breadcrumbs></pr-breadcrumbs>
 </div>
-<pr-file-list
-	[dataFolder]="someFolder"
-	(itemClicked)="showFolder($event)"
-></pr-file-list>
 <div class="share-preview-content" [ngClass]="{ navigating: isNavigating }">
-	<router-outlet
-		(activate)="subscribeToItemClicks($event)"
-		(deactivate)="unsubscribeFromItemClicks()"
-	></router-outlet>
+	@if (true) {
```

Comment:
Is there any situation where the `else` clause would happen here?

(basically: what's the point of this `if`)